charts/tyk-gateway/values.yaml

```diff
@@ -0,0 +1,133 @@
+  # addrs:
+  #   - redis.tyk.svc.cluster.local:6379
+  # This is the the DNS name of the redis as set when using Bitnami
+  #   - tyk-redis-master.tyk.svc.cluster.local:6379
+
+  # Redis password
+  # If you're using Bitnami Redis chart please input your password in the field below
+  # pass: ""
+
+  # Enables SSL for Redis connection. Redis instance will have to support that.
+  # Default value: false
+  # useSSL: true
+
+  # The enableCluster value will allow you to indicate to Tyk whither you are
+  # running a Redis cluster or not.
+  # Default value: false
+  # enableCluster: true
+
+  # By default the database index is 0. Setting the database index is not
+  # supported with redis cluster. As such, if you have enableCluster: true,
+  # then this value should be omitted or explicitly set to 0.
+  # storage:
+  #  database: 0
+
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+# The hostname to bind the Gateway to.
+hostName: tyk-gw.local
+
+# When true, sets the gateway protocol to HTTPS.
+tls: false
+
+securityContext:
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  allowPrivilegeEscalation: false
+  privileged: false
+
+replicaCount: 1
+containerPort: 8080
+image:
+  repository: tykio/tyk-gateway
+  tag: {}
+  pullPolicy: Always
+service:
+  type: NodePort
+  port: 8080
+  externalTrafficPolicy: Local
+  annotations: {}
+control:
+  enabled: false
+  containerPort: 9696
+  port: 9696
+  type: ClusterIP
+  annotations: {}
+
+ # Creates an ingress object in k8s. Will require an ingress-controller and
+ # annotation to that ingress controller.
+ingress:
+  enabled: false
+
+  # For Kubernetes >= 1.18 you should specify the ingress-controller via the field ingressClassName
+  # ingressClassName: nginx
+
+  annotations: {}
+  #  kubernetes.io/ingress.class: nginx
+  #  kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - host: tyk-gw.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+tolerations:
+  - key: node-role.kubernetes.io/master
+    effect: NoSchedule
+affinity: {}
+extraEnvs: []
+extraMounts: []
+  # - name: secret-files
+  #   mountPath: /mnt/secrets
+  #   subPath: ""
+  #   secretName: secret-files
+  #   readOnly: true
```

Comment:
Since it's going to be user provided secret, don't need all these values. Just `name` and `mountPath` will suffice.